packages/seo-integration/src/social/containers/facebook-container.js

```diff
@@ -12,29 +12,32 @@ import { useReplacementVariables } from "../../hooks/useReplacementVariables";
  * @returns {JSX.Element} A wrapped Facebook editor component.
  */
 const FacebookEditorContainer = ( { as: Component, ...restProps } ) => {
-	const title = useSelect( select => select( SEO_STORE_NAME ).selectFacebookTitle() );
-	const description = useSelect( select => select( SEO_STORE_NAME ).selectFacebookDescription() );
+	const facebookTitle = useSelect( select => select( SEO_STORE_NAME ).selectFacebookTitle() );
+	const facebookDescription = useSelect( select => select( SEO_STORE_NAME ).selectFacebookDescription() );
 	const imageData = useSelect( select => select( SEO_STORE_NAME ).selectFacebookImage() );
```

Comment:
Perhaps `facebookImage` to stay in line with the above two variable name changes?